pkg/apis/core/types.go

```diff
@@ -1556,6 +1559,57 @@ type CSIPersistentVolumeSource struct {
 	NodePublishSecretRef *SecretReference
 }
 
+// Represents a source location of a volume to mount, managed by an external CSI driver
+type CSIVolumeSource struct {
+	// Driver is the name of the driver to use for this volume.
+	// Required.
+	Driver string
+
+	// VolumeHandle is the unique volume name returned by the CSI volume
+	// plugin’s CreateVolume to refer to the volume on all subsequent calls.
+	// Required.
+	VolumeHandle string
+
+	// Optional: The value to pass to ControllerPublishVolumeRequest.
+	// Defaults to false (read/write).
+	// +optional
+	ReadOnly bool
+
+	// Filesystem type to mount.
+	// Must be a filesystem type supported by the host operating system.
+	// Ex. "ext4", "xfs", "ntfs".
+	// +optional
```

Comment:
"Default is auto-detect.  If not formatted the FS will be chosen by the node." ?

Reply:
> "Default is auto-detect. If not formatted the FS will be chosen by the node."

I see this quoted, Where is this from ?
Also,  for CSI, fstype is passed as-is delegating its interpretation to the CSI driver.

Reply:
I meant you need to document what the default is when a user doesn't specify, and that the default shouldn't be defined as a particular FS but as "the system will choose for you".

Reply:
Will fix